storage/innobase/include/trx0types.h

```diff
@@ -65,6 +65,15 @@ enum trx_state_t {
 	TRX_STATE_COMMITTED_IN_MEMORY
 };
 
+/** Transaction bulk insert operation (trx_t::bulk) */
```

Comment:
I believe that `@see trx_t::bulk` should generate a hyperlink in Doxygen output.